packages/x-charts/src/ChartsLegend/chartsLegend.types.ts

```diff
@@ -9,14 +11,59 @@ export interface LegendItemParams {
    * The label displayed in the legend
    */
   label: string;
+}
+
+export interface LegendItemConfig
```

Comment:
For me `Config` is something the developer can interact with. Those elements are props passed to the legend items. That's why I called them `Params` like "parameters of the rendering function"